packages/cli/src/commands/fix/command.ts

```diff
@@ -0,0 +1,150 @@
+    []
+  )
+  .option(
+    '-f, --format <format>',
+    'report format separated by comma, e.g. -f json,sarif,md,sonarqube',
+    parseCommaSeparatedList,
+    ['sarif']
+  )
+  .option('-d, --dryRun', `do nothing; only show what would happen`, false)
+  .addOption(
+    new Option('--rootPath <project base path>', '-- HIDDEN LOCAL DEV TESTING ONLY --')
+      .default(process.cwd())
+      .argParser(() => process.cwd())
+      .hideHelp()
+  )
+  .addOption(
+    new Option('--skipChecks', '-- HIDDEN LOCAL DEV TESTING ONLY SKIPS PREFLIGHT CHECKS --')
+      .default(false)
+      .argParser(() => false)
+      .hideHelp()
+  )
+  .action(async (src: string, options: FixCommandOptions) => {
+    await fix(src, options);
+  });
+
+async function fix(src: string, options: FixCommandOptions): Promise<void> {
+  winstonLogger.info(`@rehearsal/fix ${version?.trim()}`);
+  if (options.graph && !options.deps && !options.devDeps) {
+    console.warn(
+      `Passing --graph without --deps, --devDeps, or both results in only analyzing the local files in the package.`
+    );
+  }
+
+  if (!options.graph && options.devDeps) {
+    throw new Error(`'--devDeps' can only be passed when you pass --graph`);
+  }
+
+  if (!options.graph && options.deps) {
+    throw new Error(`'--deps' can only be passed when you pass --graph`);
+  }
+
+  if (!options.graph && options.ignore.length > 0) {
+    throw new Error(`'--ignore' can only be passed when you pass --graph`);
+  }
+
+  if (!src) {
+    throw new Error(`@rehearsal/fix: you must specify a package or path to move`);
+  }
+
+  // grab the child fix tasks
+  const { convertTask, initTask } = await loadFixTasks();
+  const { graphOrderTask } = await loadGraphTasks();
+
+  // source with a direct filepath ignores the migration graph
+  const tasks = options.graph
+    ? [
+        initTask(src, options),
+        graphOrderTask(src, {
+          rootPath: options.rootPath,
+          devDeps: options.devDeps,
+          deps: options.deps,
+          ignore: options.ignore,
+        }),
+        convertTask(options),
+      ]
+    : [initTask(src, options), convertTask(options)];
+
+  DEBUG_CALLBACK(`tasks: ${JSON.stringify(tasks, null, 2)}`);
+  DEBUG_CALLBACK(`options: ${JSON.stringify(options, null, 2)}`);
+
+  try {
+    await new Listr(tasks, {
+      concurrent: false,
+    }).run();
+  } catch (e) {
+    if (e instanceof Error) {
+      winstonLogger.error(`${e.message + '\n' + (e.stack || '')}`);
+    }
+  }
+}
+
+async function loadFixTasks(): Promise<FixTasks> {
+  return await import('./tasks/index.js').then((m) => {
+    const { initTask, convertTask } = m;
+
+    return {
+      initTask,
+      convertTask,
+    };
+  });
+}
+
+async function loadGraphTasks(): Promise<GraphTasks> {
+  return await import('../graph/tasks/graphOrderTask.js').then((m) => {
+    const { graphOrderTask } = m;
+
+    return {
+      graphOrderTask,
+    };
+  });
+}
```

Comment:
We don't need to fix this now but we should create an issue that removes this type of stuff since we assume you installed the packages before hand.